metadata-ingestion/src/datahub/ingestion/source/confluent_schema_registry.py

```diff
@@ -59,6 +61,14 @@ def __init__(
         except Exception as e:
             logger.warning(f"Failed to get subjects from schema registry: {e}")
 
+        self.field_meta_processor = OperationProcessor(
+            self.source_config.field_meta_mapping,
+            self.source_config.tag_prefix,
+            "SOURCE_CONTROL",
```

Comment:
I'm not sure what this should be. "SOURCE_CONTROL" is probably okay for dbt. Should we set the ownership source to "OTHER" or "MANUAL"here ?

Reply:
let's use `OwnershipSourceTypeClass.SERVICE`

Reply:
done